src/explore-education-statistics-admin/src/services/releaseDataFileService.ts

```diff
@@ -132,11 +132,14 @@ const releaseDataFileService = {
     data.append('metaFile', metadataFile);
     data.append('replacingFileId', replacingFileId ?? '');
 
-    const file = await client.post<DataFileInfo>(`/releaseVersions/data`, data);
+    const file = await client.post<DataFileInfo>(
+      `/releaseVersions/replacement-data`,
```

Comment:
This can use simple hyphens rather than backticks if there's no string tokenisation going on.  I realise it was like this when you found it!